build/baremetalpi/Makefile

```diff
@@ -0,0 +1,47 @@
+#
+# Makefile
```

Comment:
Would you like help porting this to CMake?

Reply:
Well, that would be awesome but there's some issues: that Makefile includes a lot of stuff from circle .mk files. So first of all, it won't be easy, and second, if we "copy" over some of that stuff, we diverge and stay behind in case circle adjust something. I think we'll better stay like this, at least for now (i'm still making some adjustment).

Btw have you tried the port maybe? :)

Reply:
Ah, I see...
```
include $(CIRCLEHOME)/Rules.mk
```